cmd/scan.go

```diff
@@ -71,7 +71,7 @@ var (
 					util.PrintInfo("Ecosystems detected:", strings.Join(ecosystems, ","))
 					util.PrintInfof("Scanning %v packages for vulnerabilities...\n", len(purls))
 					util.PrintInfo("Vulnerability Provider:", provider.Info(), "\n")
-					s.Suffix = fmt.Sprintf(" Fetching vulnerability data from %s", providerName)
```

Comment:
This change appears to trigger a bug for me.

I'd suggest keeping the PR just to the Snyk provider, and then opening a separate issue/PR around this if needed.

For me, this triggered the following behaviour.

```
| Fetching vulnerability data from osv
/ Fetching vulnerability data from osv
- Fetching vulnerability data from osv
\ Fetching vulnerability data from osv
| Fetching vulnerability data from osv
/ Fetching vulnerability data from osv
- Fetching vulnerability data from osv
```